examples/vg-specs/area.vg.json

```diff
@@ -33,6 +33,9 @@
                     ],
                     "fields": [
                         "count"
+                    ],
+                    "as": [
```

Comment:
1. This makes the code way more maintainable
2. This makes the output code explicit about field names, which is not necessary a bad thing.